DevSkim-DotNet/Microsoft.DevSkim.CLI/Commands/AnalyzeCommand.cs

```diff
@@ -288,52 +309,52 @@ string TryRelativizePath(string parentPath, string childPath)
 
         private int RunFileEntries(IEnumerable<FileEntry> fileListing, Languages devSkimLanguages)
         {
-            DevSkimRuleSet devSkimRuleSet = opts.IgnoreDefaultRules ? new() : DevSkimRuleSet.GetDefaultRuleSet();
-            if (opts.Rules.Any())
+            DevSkimRuleSet devSkimRuleSet = _opts.IgnoreDefaultRules ? new() : DevSkimRuleSet.GetDefaultRuleSet();
+            if (_opts.Rules.Any())
             {
-                foreach (string path in opts.Rules)
+                foreach (string path in _opts.Rules)
                 {
                     devSkimRuleSet.AddPath(path);
                 }
                 DevSkimRuleVerifier devSkimVerifier = new DevSkimRuleVerifier(new DevSkimRuleVerifierOptions()
                 {
-                    LanguageSpecs = devSkimLanguages
-                    //TODO: Add logging factory to get validation errors.
+                    LanguageSpecs = devSkimLanguages,
+                    LoggerFactory = _logFactory
```

Comment:
should this be getting _logger instead of _logFactory?

Reply:
https://github.com/microsoft/ApplicationInspector/blob/71042add5dba69e47d03de953e3bcf67d126b81b/AppInspector.RulesEngine/RulesVerifier.cs#L32

Reply:
thanks! i see now.